modules/tinymce/src/plugins/table/main/ts/api/Settings.ts

```diff
@@ -22,6 +22,8 @@ const defaultStyles = {
   'width': '100%'
 };
 
+const columnGroupDefault = false;
```

Comment:
Why make this a separate variable when it's not used elsewhere? This really could just be inlined.